fitbert_test_data/get_phrase.py

```diff
@@ -0,0 +1,90 @@
+import re
+
+dictionary = {
```

Comment:
is this format from wordnet?

Reply:
There are 2 reasons to write custom mapping:

1. Non compatible/obsolete alternatives:

- spacy - only noun chunks

- wordnet - different than spacy pos tag system
https://www.guru99.com/pos-tagging-chunking-nltk.html
https://spacy.io/api/annotation

- textacy - obsolete as we had the following changes:
https://universaldependencies.org/v2/postags.html
CONJ -> CCONJ
In fact also all SCONJ has been mapped to CCONJ (probe on 90k corpus).

2. Possibility to add statistical gaps in the near future: 

Custom mapping allows to find longest common pos tag sequences using Sequence Matcher (https://docs.python.org/2/library/difflib.html).
-> given crowd-sourced examples of fill in the blanks we can find statistically probable places for gaps.
This piece of code has been implemented (and in fact took >75% of coding time), but since no open source examples are available atm - skipped.

Reply:
Can you walk me through this on a call tomorrow?

Reply:
Sure.